requirements.txt

```diff
@@ -10,3 +10,4 @@ BeautifulSoup4
 mock
 xmltodict
 rtyaml
+future
```

Comment:
I don't think this line is need for this PR, right?

Reply:
@JoshData that's correct, snuck into this commit accidentally. I can remove to keep it clean and focused.

Reply:
Should be fixed now